src/lib/viewers/controls/controls-root/ControlsRoot.tsx

```diff
@@ -67,7 +82,32 @@ export default class ControlsRoot {
         this.controlsEl.classList.remove('bp-is-hidden');
     }
 
+    shouldForceShow(): boolean {
+        if (this.wasClosedByUser.tooltipFlowAnnotationsExperience) {
+            return false;
+        }
+
+        if (!this.experiences.tooltipFlowAnnotationsExperience) {
+            return false;
+        }
+
+        return this.experiences.tooltipFlowAnnotationsExperience.canShow;
```

Comment:
Is there a way this logic could be enhanced to scale to support more experiences?

Reply:
Let me know what you think of my solution that can be used by more experiences once I push my changes!